test/common/storagelib.py

```diff
@@ -90,6 +90,9 @@ def add_loopback_disk(self, size=50, name=None):
         self.machine.execute("partprobe '%s'" % dev)
         # right after unmounting the device is often still busy, so retry a few times
         self.addCleanup(self.machine.execute, f"umount {dev} || true; rm $(losetup -n -O BACK-FILE -l {dev}); until losetup -d {dev}; do sleep 1; done", timeout=10)
+        # check if umount and detaching loopback  device were successul.
+        self.addCleanup(self.machine.execute, f"cat /proc/mounts | ! grep '^{dev}'")
```

Comment:
Let's please do this more robustly. I always hated the `|| true`, as that hides errors. Instead, let's check with `findmnt` if `dev` is actually mounted anywhere, and then run umount without `|| true`.

Reply:
Not sure if it help, beucase more troubles caused me, that, that FM is busy, so umount fail. and thus cannot umount, then removing loopback fails,  than device is mounted more times, especially there is umount of device, not mountpoint, so it should umount all mount points I think.

Reply:
Right, I like the findmnt approach here. It now [fails a lot](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19910-20240130-201809-31ffce59-centos-8-stream-storage/log.html) ("umount: bad usage"), but it's the general spirit.